packages/docusaurus-theme-classic/src/theme/NavbarItem/DropdownNavbarItem.tsx

```diff
@@ -88,6 +88,7 @@ function DropdownNavbarItemDesktop({
         'dropdown--show': showDropdown,
       })}>
       <NavLink
+        href="#"
```

Comment:
This isn't actually fully correct—some dropdowns are links themselves, and adding an `href` would make it override the `to` property. You can observe this on the deploy preview where the docs version dropdown is no longer taking you to the doc homepage of the version (compare with the production website).

Reply:
Oops, right, I hadn't noticed that. So actually this is only a problem if the top-level item is not a link

Reply:
What do you think, should I push a fix for this to this branch and we handle the larger accessibility improvements around dropdowns separately, or do you want to batch them in a separate PR?

Reply:
I'd prefer we do it all here. I'll take care of it. Thanks for bringing it up, we do have some a11y issues

Reply:
Sounds good—check my message below as well :) 

Also I think this specific issue can just be fixed with:

```tsx
href={props.to ? undefined : "#"}
```

(similar to how the `onClick` is handled)

Reply:
(☝️ just tried this to test it on the deploy preview)

Reply:
Looks like it works as expected, resolving